src/EnergyPlus/CurveManager.cc

```diff
@@ -2802,10 +2802,10 @@ namespace CurveManager {
     bool CheckCurveDims(EnergyPlusData &state,
                         int const CurveIndex,
                         std::vector<int> validDims,
-                        std::string routineName,
-                        std::string objectType,
-                        std::string objectName,
-                        std::string curveFieldText)
+                        std::string_view routineName,
+                        const std::string &objectType,
+                        const std::string &objectName,
+                        const std::string &curveFieldText)
```

Comment:
Clang tidy warnings about const arguments in headers were cleaned up here while I was resolving conflicts.